server/libs/data_dump/volunteers.js

```diff
@@ -0,0 +1,354 @@
+export default [
```

Comment:
I'm confused as to how this PR is adding this file. Doesn't this file already exist?